docs/user/commercial/privacy-level.rst

```diff
@@ -1,20 +1,41 @@
-.. TODO: This is a super weird page..
+Privacy Levels
+--------------
 
-:orphan:
+.. include:: /shared/admonition-rtd-business.rst
 
-Project privacy level
----------------------
+Privacy levels allow you to set the visibility of your project and its versions.
+This allows you to specify what information is available to the public and what is private.
+Privacy is generally controlled at the level of the version,
```

Comment:
```suggestion
Privacy can be controlled at the level of the project and each version,
```

I'm making this distinction here because the main content of the page talks about "Project privacy" and "Version privacy"